scripts/bootstrap/install.py

```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env python3
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#   "zstandard==0.22.0",
+# ]
+# ///
```

Comment:
What is this comment?
Does pipx support this?

Reply:
It seems I used old pipx. Now I can ran it.
I understand this is [PEP 723](https://peps.python.org/pep-0723/).